roles/bootstrap-os/tasks/bootstrap-fedora.yml

```diff
@@ -17,5 +17,8 @@
 
 - name: Install required python packages
   dnf:
-    name: libselinux-python
+    name: "{{ item }}"
```

Comment:
That is not necessary as httplib2 is installed via system packages on fedora

Reply:
I have pip installing httplib2 as https://github.com/kubernetes-incubator/kubespray/issues/1032 specifically seems to impact the fedora os family.